relay/exchange/exchange_db.py

```diff
@@ -5,7 +5,7 @@
 from sqlalchemy.orm import sessionmaker
 
 from .order import Order
-from eth_utils import force_bytes
+from relay.compat import force_bytes
```

Comment:
Do we still need this? This was because of a bug in the old web3, so it should not be necessary anymore.

Reply:
right. that's one of the things I still wanted to look into. The way I understood the changelog of web3 these calls shouldn't be necessary anymore.